cmd/crc/cmd/start.go

```diff
@@ -78,6 +74,14 @@ func runStart(ctx context.Context) (*machine.StartResult, error) {
 	}
 
 	client := newMachine()
+	exists, _ := client.Exists()
+
+	if !exists {
```

Comment:
Fine for me. This code you wrote should also be duplicated in the daemon API.

I am tempted to move this in the machine.Start function.

Reply:
The problem is that the prefflights do not run from the start... and the start already detects this situation.